src/api/workspace.rs

```diff
@@ -1128,6 +1135,56 @@ async fn delete_published_collab_comment_handler(
   Ok(Json(AppResponse::Ok()))
 }
 
+#[derive(Deserialize)]
+struct GetReactionQuery {
+  comment_id: Option<Uuid>,
+}
+
```

Comment:
`GetReactionQuery` can be shared as a common param structure for `client-api`. it can be passed in from the client side in the reqwest `query` method.